homeassistant/components/onewire/onewire_entities.py

```diff
@@ -76,16 +76,14 @@ def _read_value_ownet(self):
     def update(self):
         """Get the latest data from the device."""
         value = None
-        value_read = False
         try:
-            value_read = self._read_value_ownet()
+            self._value_raw = self._read_value_ownet()
```

Comment:
Is it OK that self._value_raw used to be a float? Old code was "self._value_raw = float(value_read)".   This was only exposed via device_state_attributes so presumably that is OK, but just thought it was worth asking for sure.

Reply:
You are correct - I've updated to avoid a (potentially) breaking change.